packages/cli/bin/console

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env php
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\Cli;
+
+use T3Docs\Cli\Command\Console;
+use Symfony\Component\Console\Application;
+
+$vendorDir = dirname(__DIR__) . '/../../vendor';
+$autoloadDirectory = $vendorDir . '/autoload.php';
+if (file_exists($autoloadDirectory)){
```

Comment:
Why do you think this is needed? If the autoload.php is not available in the vendor folder then also this application is not available? Or do I overlook something?

Reply:
(This mainly stems from the reason above, I copy+pasted it from the bin/guides runner)